codegen/plugins/resolver/plugin.go

```diff
@@ -0,0 +1,62 @@
+package resolver
+
+import (
+	"context"
+
+	"github.com/99designs/gqlgen/codegen"
+	"github.com/99designs/gqlgen/graphql"
+	"github.com/pkg/errors"
+	"github.com/vektah/gqlparser/ast"
+)
+
+type Plugin struct{}
+
+var plugin = &Plugin{}
+
+func (p *Plugin) Schema(cfg *codegen.Config) (*ast.Source, error) {
+	return &ast.Source{
+		Name:  "resolver plugin",
```

Comment:
I wonder if this should be done by the caller and this function can just return the schema string?